libs/components/src/cascader/vl-cascader.component.ts

```diff
@@ -0,0 +1,311 @@
+        }
+    };
+
+    private navigateBack() {
+        if (this.isLoading) return;
+        if (this.navigationStack.length > 0) {
+            this.slidingOut = true;
+            this.updateNavigationStack();
+        }
+        this.requestUpdate();
+    }
+
+    jumpToNavigation(index: number): void {
+        if (this.isLoading) return;
+        const length = this.navigationStack.length;
+        const regressionNumber = length - index;
+        if (regressionNumber > 0) {
+            Array.from({ length: regressionNumber }).map(() => {
+                this.navigateBack();
+            });
+        }
+    }
+
+    renderBreadcrumbHome = () => {
+        return html`
+            <li class="vl-breadcrumb__list__item">
+                <span class="vl-breadcrumb__list__item__separator" aria-hidden="true"></span>
+                <span
+                    is="vl-icon"
+                    data-vl-icon="places-home"
+                    class="vl-breadcrumb__list__item__cta"
+                    @click=${() => this.jumpToNavigation(0)}
+                ></span>
+            </li>
+        `;
+    };
+
+    renderBreadcrumbItem = ({ label, index }: { label: string; index: number }): TemplateResult => {
+        const historyLength = this.breadCrumbHistory?.length;
+        const isCurrentLevel = historyLength === index;
+        const breadCrumbItemClasses = {
+            'vl-breadcrumb__list__item': true,
+            'vl-breadcrumb__list__item--deactivated': isCurrentLevel,
+        };
+        return html`
+            <li class=${classMap(breadCrumbItemClasses)}>
+                <span class="vl-breadcrumb__list__item__separator" aria-hidden="true"></span>
+                <span class="vl-breadcrumb__list__item__cta" @click=${() => this.jumpToNavigation(index)}
+                    >${label}</span
+                >
+            </li>
+        `;
+    };
+
+    renderBreadcrumb(): TemplateResult {
+        const historyLength = this.breadCrumbHistory?.length;
+        const hasBreadcrumbPlaceholderSlot = getNodesForSlot(this, CASCADER_SLOTS.BREADCRUMB_PLACEHOLDER);
+        return html`
+            ${historyLength && !this.hideBreadcrumb
+                ? html`
+                      <nav aria-label="U bent hier: " class="vl-breadcrumb">
+                          <ol class="vl-breadcrumb__list">
+                              ${this.renderBreadcrumbHome()} ${this.breadCrumbHistory?.map(this.renderBreadcrumbItem)}
+                          </ol>
+                      </nav>
+                  `
+                : hasBreadcrumbPlaceholderSlot
+                ? html`
+                      <nav class="vl-breadcrumb-placeholder">
+                          <slot name="breadcrumb-placeholder"></slot>
+                      </nav>
+                  `
+                : nothing}
+        `;
+    }
+
+    static getTemplateFunctionForType(item: NodeData, cascader: VlCascaderComponent): TemplateResult | void {
```

Comment:
Je zou deze methode in de utils kunnen zetten ipv een static methode te voorzien.